### PR TITLE
fix: merkle root format

### PR DESCRIPTION
**Pull request by 0xArdy**

### CODE REVIEW

**Review by 0xDiscotech — APPROVED** [review]

**Review by 0xtiti — APPROVED** [review]

File: src/utils/misc.ts [hunk 1]
```diff
@@ -22,3 +26,29 @@ export const formattedDate = (timestamp: number): string => {
...
+    parseAbiParameters('uint256 nullifier_hash'),
+    nullifier_hash as Hex,
+  );
+  const [decodedProof] = decodeAbiParameters(parseAbiParameters('uint256[8] proof'), proof as Hex);
+  const proofData = encodePacked(
+    ['uint256', 'uint256', 'uint256[8]'],
+    [decodedMerkleRoot, decodedNullifierHash, decodedProof],
+  );
+  return proofData;
+}
```
Comment (agusduha) [inline src/utils/misc.ts:54]: Why using camel case here?

Comment (0xShaito) [reply src/utils/misc.ts:54]: snake case*

File: src/types/Data.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+export interface ISuccessResult {
+  merkle_root: string;
+  nullifier_hash: string;
```
Comment (0xShaito) [inline src/types/Data.ts:3]: Do we use snake case?

**Review by 0xShaito — APPROVED** [review]

File: src/utils/misc.ts (hunk 1 above)
Comment (0xtiti) [reply src/utils/misc.ts:54]: https://docs.worldcoin.org/reference/idkit#types due to how it comes from sdk

Comment (0xtiti) [reply src/utils/misc.ts]: [d08a1a5](https://github.com/defi-wonderland/world-id-goats-dao-ui/pull/29/commits/d08a1a5a96a0128a2f360a37e173da21c557b137)

**Review by agusduha — APPROVED** [review]

**Review by 0xShaito — APPROVED** [review]

**Pull request merged**